### ISSUE TITLE
Does apiman-studio need i18n

### ISSUE THREAD

**Issue by msavy** [apiman/apiman-studio#22]
- Do we need it? 
- What's the best approach for this app?
- Should we use the [Angular 2](https://angular.io/docs/ts/latest/cookbook/i18n.html) i18n toolset. It seems vastly better than Angular v1's stuff.

**Comment by Yuutakasan** [comment]
I want translation function by all means. Japanization cooperation is possible.

**Comment by EricWittmann** [comment]
Unfortunately right now we do not have i18n support in Apicurio.  I'm hopeful that a re-working of the UI using Patternfly 4 (a process that will hopefully be happening in 2019) will help with this.  But there's no timeframe yet on it.

**Comment by deka** [comment]
Please add i18n support.

**Comment by Yuutakasan** [comment]
@EricWittmann 
I am planning to make apicurio multilingual this year.
Do you consider recreating the view layer in the future when you use PatternFly4?

If I make a multilingualization of the current view layout this year and send a merge request, would you be able to commit it?

**Comment by EricWittmann** [comment]
Hi @Yuutakasan - we are currently working on replacing the UI with a React/PatternFly 4 implementation.  That project is ongoing here:  https://github.com/apicurio/apicurio-studio-react-poc

I would definitely recommend looking there for how you could contribute i18n support rather than do anything in the current generation UI.

**Comment by Yuutakasan** [comment]
@EricWittmann 
I was doing something useless if I was working on it. Thanks for the information.

**Comment by EricWittmann** [comment]
@dlabaj Could you provide any insight/feedback on the best way @Yuutakasan could perhaps help with i18n support in the new Apicurio Studio React POC?

**Comment by Yuutakasan** [comment]
@EricWittmann 
If I can release the internationalization program by November 2020, I think I can develop Japanese translation and internationalization and send a Pull Request from August.

**Comment by EricWittmann** [comment]
A Japanese translation would be fantastic, of course!  Hopefully PF4 has good i18n support that we can leverage.  @dlabaj any thoughts on that?

**Comment by Yuutakasan** [comment]
@EricWittmann 
If you can provide us with the language files, we can translate them for you, and once the project starts, we may be able to assign an engineer to develop modifications to the view logic. Unfortunately, even if the project doesn't start, I can help with the translation.

**Comment by turutosiya** [comment]
I'm looking for i18n support in Apicurio front-end and finally found this issue.
But unfortunately, it seems that `apicurio-studio-react-poc` has no commit recently...
Is there any progress with this?

**Comment by EricWittmann** [comment]
Agree the Studio is missing i18n.  No current progress to report on that issue.

**Comment by Yuutakasan** [comment]
@EricWittmann 
If we code Apicurio Studio to support multiple languages, is it possible to include pull requests?

**Comment by EricWittmann** [comment]
Yes for sure - I would suggest we start small and get just a few translations so we can work out the details though, before you spend too much time doing the actual string translations.

**Comment by Yuutakasan** [comment]
Thank you. I'll send a small pull request.

**Comment by turutosiya** [comment]
@EricWittmann I've made a pull request, would you please check?
https://github.com/Apicurio/apicurio-studio/pull/1391

**Comment by EricWittmann** [comment]
Hi @turutosiya - I skimmed it quickly earlier today but then got pulled into some other work.  I'll be taking a closer look tomorrow.  Thanks for the POC!

**Comment by Yuutakasan** [comment]
@EricWittmann ( @turutosiya  )
While discussing yesterday, I noticed that when we do multilingualization on the Anguler side, if the message is directly embedded in Java, such as apicurio-data-models (e.g. OpenAPI spec validation message), we can't resolve it in the frontend part, so we can't make it multilingual.

**Comment by turutosiya** [comment]
@EricWittmann The previous PR contains unnecessary commits in it. So, I create a new PR. It's clean:
https://github.com/Apicurio/apicurio-studio/pull/1463

I'll continue creating language resources.

**Comment by turutosiya** [comment]
https://github.com/Apicurio/apicurio-studio/pull/1466
